### PR TITLE
Add Trino project info to Writing Day

### PR DESCRIPTION
**Pull request by rosewms**

### CODE REVIEW

File: docs/conf/portland/2021/writing-day.rst [hunk 1]
```diff
@@ -24,8 +24,30 @@ The `Good Docs Project <https://thegooddocsproject.dev/>`_ is an open source com
 - **Easy button for docs setup** - One of the biggest hurdles to creating good documentation is setting up your initial documentation toolchain. Come collaborate with our docs tool experts who are working on creating a Hugo/Docsy repository that can be easily cloned to set up a new documentation system for a project.
 - **Industrial fiction** - Our goal is to provide good examples of our project's templates being used in an actual project. To accomplish this goal, we're creating our own fake open source project that can be used to generate documentation examples. Come help us build both the world and the documentation for The Chronologue, a fake open source tool that helps future time travelers sync their clocks across multiple dimensions.
 
+Trino project
+^^^^^^^^^^^^^
+
+`Trino <https://trino.io>`_ is a fast distributed SQL query engine for big data
+analytics. We’re excited to join you Write the Docs, this is our first `Writing Day <https://trino.io/blog/2021/04/14/wtd-writing-day.html>`_! We’ll have Trino
```
Comment (mosabua) [inline docs/conf/portland/2021/writing-day.rst]: to join Write the Docs. This our first

File: docs/conf/portland/2021/writing-day.rst [hunk 2]
```diff
@@ -24,8 +24,30 @@ The `Good Docs Project <https://thegooddocsproject.dev/>`_ is an open source com
...
+enthusiasts available to help you get started, we even made `a video to help you get started <https://youtu.be/yseFM3ZI2ro>`_.
```
Comment (mosabua) [inline docs/conf/portland/2021/writing-day.rst]: contributors

Comment (mosabua) [inline docs/conf/portland/2021/writing-day.rst]: get started. We even

File: docs/conf/portland/2021/writing-day.rst [hunk 3]
```diff
@@ -24,8 +24,30 @@ The `Good Docs Project <https://thegooddocsproject.dev/>`_ is an open source com
...
+
+Here’s a few ways you can contribute to the Trino open source project:
```
Comment (mosabua) [inline docs/conf/portland/2021/writing-day.rst]: Here are a few ways

File: docs/conf/portland/2021/writing-day.rst [hunk 4]
```diff
@@ -24,8 +24,30 @@ The `Good Docs Project <https://thegooddocsproject.dev/>`_ is an open source com
...
+
+- `Review our doc readme <https://github.com/trinodb/trino/tree/master/docs>`_, 
+   perfect for first time contributors.
+- `Improve our docs <https://trino.io/docs/current/>`_, we made an `issues filter <https://github.com/trinodb/trino/issues?q=is%3Aopen+label%3Adocs+label%3A%22good+first+issue%22>`_ to help you find ways to contribute. 
+   - Edit an existing document, perfect for
```
Comment (mosabua) [inline docs/conf/portland/2021/writing-day.rst]: Why the short lines here in the bullet list?

File: docs/conf/portland/2021/writing-day.rst [hunk 5]
```diff
@@ -24,8 +24,30 @@ The `Good Docs Project <https://thegooddocsproject.dev/>`_ is an open source com
...
+   first time contributors. 
+   - Add terms to our glossary, perfect for first time
+   contributors. 
+   - Add missing feature info in existing docs.
+- Test our docs and provide feedback.
+    - `Learn Trino SQL on Docker <https://www.youtube.com/watch?v=y58sb9bW2mA>`_, we made a video to help folks test,
```
Comment (mosabua) [inline docs/conf/portland/2021/writing-day.rst]: test.

File: docs/conf/portland/2021/writing-day.rst [hunk 6]
```diff
@@ -24,8 +24,30 @@ The `Good Docs Project <https://thegooddocsproject.dev/>`_ is an open source com
...
+
+We love new community members, join us on the `Trino Slack <https://trino.io/slack.html>`_ on the `#documentation channel <https://trinodb.slack.com/messages/C01TEP0HJTH>`_ and `favorite the Trino project <https://github.com/trinodb/trino>`_ on GitHub.
```
Comment (mosabua) [inline docs/conf/portland/2021/writing-day.rst]: members. Join us

**Review by mosabua — CHANGES_REQUESTED** [review]
A couple of changes needed.

File: docs/conf/portland/2021/writing-day.rst (hunk 4 above)
Comment (rosewms) [reply docs/conf/portland/2021/writing-day.rst]: It's supposed to be the bullets under the "Improve our docs" section.

**Review by mosabua — APPROVED** [review]
Looks good to me now. Rendering should be validated .. I did not build locally ;-)

**Comment by plaindocs** [comment]
The build is here, bullets are a little wonky.

https://writethedocs-www--1489.org.readthedocs.build/conf/portland/2021/writing-day/#trino-project

**Comment by rosewms** [comment]
@plaindocs for the life of me, I can't figure out why the formatting is off on those bullets. I followed the same formatting as the project above...

File: docs/conf/portland/2021/writing-day.rst [hunk 7]
```diff
@@ -24,8 +24,25 @@ The `Good Docs Project <https://thegooddocsproject.dev/>`_ is an open source com
...
+
+Here are a few ways you can contribute to the `Trino open source docs <https://trino.io/docs/current/>`_:
+
+- Review our `doc readme <https://github.com/trinodb/trino/tree/master/docs>`_ perfect for first time contributors.
+- Improve our docs, we made an `issues filter <https://github.com/trinodb/trino/issues?q=is%3Aopen+label%3Adocs+label%3A%22good+first+issue%22>`_ to help you find ways to contribute. 
+   - Edit an existing document, perfect for first time contributors. 
+   - Add terms to our glossary, perfect for first time contributors. 
+   - Add missing feature info in existing docs.
+- Test our docs and provide feedback.
+    - `Learn Trino SQL on Docker <https://www.youtube.com/watch?v=y58sb9bW2mA>`_, we made a video to help folks test.
```
Comment (plaindocs) [inline docs/conf/portland/2021/writing-day.rst]:
```suggestion
- Review our `doc readme <https://github.com/trinodb/trino/tree/master/docs>`_ perfect for first time contributors.
- Improve our docs, we made an `issues filter <https://github.com/trinodb/trino/issues?q=is%3Aopen+label%3Adocs+label%3A%22good+first+issue%22>`_ to help you find ways to contribute. 

  - Edit an existing document, perfect for first time contributors. 
  - Add terms to our glossary, perfect for first time contributors. 
  - Add missing feature info in existing docs.

- Test our docs and provide feedback.
  
  - `Learn Trino SQL on Docker <https://www.youtube.com/watch?v=y58sb9bW2mA>`_, we made a video to help folks test.
```

**Review by plaindocs — APPROVED** [review]
@rosewms I merged a fix

**Pull request merged**